### PR TITLE
Add  new module, clock map, and SQLX support

### PR DESCRIPTION
**Pull request by FMassin**
SQLX compute (P)PSD continuously. If this is available, there is no
point in wasting time computing PPSD with obspy. This is adding a
`PSDs`python object for handling PSD from SQLX in a very similar manner
to the original notebook SeismoSocialDistancing.ipynb.

For now the interface with SQLX is a rather rough system call to SQLX's
script `exPSDhour` via an ssh connection. None of this is optimised or
properly included in the class.

There is also a new very sketchy notebook demonstrating ho to use this
new python object.

It now take a few minutes to get 3-component RMS over several months :)

### CODE REVIEW

**Comment by FMassin** [comment]
Should I leave this open for a little while ? What our working scheme for PR :) ? I'll wait from for feedback from at least @ThomasLecocq @claudiodsf ...

**Comment by FMassin** [comment]
Following uo on the nice `clockplots` added to the main notebook by
@ThomasLecocq and the growing complexity, I mean to start developing
within a module, than can be then imported in notebooks.

The module is `seismosocialdistancing.py`. I am proposing to put all
our functions there and import the module in all notebooks instead of
creating functions or long codes in notebook cells. In case it is not
obvious, while developing a new function in module, one can reload the
module with:
```
import imp
imp.reload(seismosocialdistancing)
```

I demonstrate how to in sqlxSocialDistancing.ipynb with:
- the sqlx2rms function which produce an object with PSDs and an dRMS
method.
- the hourmap function (poor name?) which makes a map of dRMS aroud the
clock.

**Comment by ThomasLecocq** [comment]
good idea !

**Comment by FMassin** [comment]
OK for merging?

**Comment by ThomasLecocq** [comment]
OK ! go :-)

File: seismosocialdistancing.py [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+        datelist = pd.date_range(date, 
+                                 date+pd.Timedelta(9, unit='D'), 
+                                 freq="30min")
+        #print(datelist)
+        # Send ssh commands to stdin
+        for date1 in datelist:
+            if date1>= end.datetime:
+                break
+            date2 = date1+pd.Timedelta(minutes=30)
+            for n in network.split(','):
```
Comment (ThomasLecocq) [inline seismosocialdistancing.py:128]: ``for (n,s,l,c) in zip(network.split(","), station.split(","), location.split(","), channel.split(",")):``

works ?

**Comment by ThomasLecocq** [comment]
but yéé, merge !

File: seismosocialdistancing.py (hunk 1 above)
Comment (FMassin) [reply seismosocialdistancing.py:128]: No. It does:
```

network='0'
station='A,B'
location=''
channel='Ω,µ'
for (n,s,l,c) in zip(network.split(","), station.split(","), location.split(","), channel.split(",")):
    print('%s.%s.%s.%s'%(n,s,l,c))
> 0.A..Ω
```
I want:
```
> 0.A..Ω
> 0.B..Ω
> 0.A..µ
> 0.B..µ
```

Comment (ThomasLecocq) [reply seismosocialdistancing.py:128]:
```python
net = network.split(",")
sta = station.split(",")
loc = location.split(",")
chan = channel.split(",")
from itertools import combination_with_replacement
for (n,s,l,c) in combination(net,sta,loc,chan):
   # do something
```

or something similar :-) anyway, not important now

Comment (FMassin) [reply seismosocialdistancing.py:128]: OK next one :)

**Pull request merged**